js/context.js

```diff
@@ -31,8 +31,8 @@ module.exports = {
       '@type': '@id',
       '@id': 'https://w3id.org/security#anchoredResource'
     },
-    contentHash: {
-      '@id': 'https://w3id.org/security#contentHash',
+    multibaseDigest: {
+      '@id': 'https://w3id.org/security#multibaseDigest',
```

Comment:
I think this better matches the naming convention used elsewhere, e.g., `publicKeyMultibase`.

```suggestion
    digestMultibase: {
      '@id': 'https://w3id.org/security#digestMultibase',
```